src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -11,6 +13,19 @@
      * числа 92, 18, 26 и 0.</p>
      */
     public int[] getAllNumbers(String text) {
-        return null;
+        ArrayList<Integer> arrayList = new ArrayList<>();
+        String[] words = text.replaceAll("[^0-9 + -]+", " ").split(" ");
+        for (int i = 0; i < words.length; i++) {
+            words[i] = words[i];
+            if (words[i].isEmpty()) {
+                continue;
+            }
+            arrayList.add(Integer.parseInt(words[i]) * 2);
+        }
+        int[] arrayWithNumbers = new int[arrayList.size()];
+        for (int i = 0; i < arrayWithNumbers.length; i++) {
+            arrayWithNumbers[i] = arrayList.get(i);
+        }
```

Comment:
try to find method which may convert list to array or you can fill this array in previous loop

Reply:
done